crud.py

```diff
@@ -22,10 +23,11 @@ async def create_usermanager_user(data: CreateUserData) -> User:
 
     await db.execute(
         """
-        INSERT INTO usermanager.users (id, name, admin, email, password)
-        VALUES (?, ?, ?, ?, ?)
+        INSERT INTO usermanager.users (id, name, admin, email, password, attrs)
```

Comment:
i would call it `extra` instead of attrs. we use extra on our payment aswell

Reply:
makes sense. 
should probably add an endpoint for the method aswell something like 
GET /api/v1/usermanager/user?admin=23452345&extra.discord_id=6734572347